### ISSUE TITLE
Doesn't seem to be working at all...

### ISSUE THREAD

**Issue by Ghozer** [jkevin/PS3EyeDirectShow#19]
HI,

Installed driver, plugged in camera - and at first Device manager simply had a red ! over it, saying "The Device cannot start" - I have eventually now got it to show as "USB Camera-B4.09.24.1" (not under Imaging devices, and nothing under Sound, Video and Game controllers) - but nothing will detect or work with it, windows 'camera' app, OBS, Skype, Discord, Chrome.... 

Can't get anything to see the camera.... the ONLY one I could get working in any fashion, was the CL driver, and it only worked with it's own 'test' app... 

W10 Pro x64 - 21H1

**Comment by minch-dev** [comment]
TL/DR: The solution that I've found back in the day is to use camera proxy app that allows multiple apps to have access to the same video source at the same time. 
You can also try using some linux distribution that already has this driver built-in natively. Like KUbuntu, for one.

CL driver (the last free version) is fine, worked for me the last time I checked in the same environment as yours. Although you can't use camera's built in microphone and a camera itself  at the same time, it used to work in win7 and xp, but windows doesn't get better over time and things tend to break to never be fixed. So basically when you try to use it in those apps the first thing they use is the mic, and the access to camera is locked.

Also it's quite possible that you might have a problem with x86/x64 driver/app mismatch.

**Comment by Ghozer** [comment]
I tried CL Driver, only works in 32bit apps or it's own test app, have to use some kinda bridge for that - which i want to avoid using as it's yet another layer to cause problems, and more delay/bloat I don't need or want :)

It's fine, i'll just invest in a real camera and stop trying to hack something, Just wanted to try and get it working to save a bit of money that I don't really have :)

**Comment by minch-dev** [comment]
Well PS3eye is more than a real camera, the market kinda doesn't offer any cams with steady frame rate under any lighting conditions for like 5 bucks. This guy right [here](https://forums.raspberrypi.com/viewtopic.php?t=193033) have managed to get it's fps as high as 187 using raspberry pi, which is pretty crazy.
